### DISCUSSION TITLE
What is a group of 🦏🦏 called?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#859] [Galaxy Trivia] [answered]
**Bonus:** How many species of 🦏🦏 are there in the world?

**Comment by ppnwt** [comment]
5 species
 Two African rhino species 
 - black rhinos
 - white rhinos
  Three are Asian rhino species
  - Indian Rhinoceros (greater one-horned rhinos), 
  - Sumatran rhinos
  - Javan rhinos.

**Comment by Carol42** [comment]
A group of rhinos is called a **crash**.

There are currently five species of Asian and African rhinoceros left in the world:
- African:
  - Black rhinos;
  - White rhinos;
- Asian:
  - Greater one-horned rhinos (aka Indian rhinos);
  - Javan rhinos;
  - Sumatran rhinos;


Three of them are critically endangered: black, Javan, and Sumatran.

**Comment by peterbaccaro** [comment]
A group of rhinoceroses is commonly referred to as a "**crash**" or a "**herd**".

There are five extant (living) species of rhinoceroses in the world. They are:

- **White Rhinoceros (Ceratotherium simum)**: There are two subspecies of white rhinoceros, the Southern white rhinoceros and the critically endangered Northern white rhinoceros.

- **Black Rhinoceros (Diceros bicornis)**: The black rhinoceros is listed as critically endangered.

- **Indian Rhinoceros (Rhinoceros unicornis)**: The Indian rhinoceros is primarily found in parts of India and Nepal and is listed as vulnerable.

- **Javan Rhinoceros (Rhinoceros sondaicus)**: The Javan rhinoceros is one of the rarest and most endangered large mammals in the world.

- **Sumatran Rhinoceros (Dicerorhinus sumatrensis)**: The Sumatran rhinoceros is also critically endangered and is one of the smallest rhinoceros species.

**Comment by ParanoidUser** [comment]
**Extra:** What is the actual color of ⚫ and ⚪ Rhinos? Why are they so named?

**Reply by PatilHarshh** [reply]
Black Rhino: The black rhinoceros  is known as the "black" rhino due to the dark coloration of its skin. However, their actual color can vary from dark brown to gray, and they are not truly black.

 White Rhino: The white rhinoceros  is called the "white" rhino, which is a misinterpretation of its name. The term "white" is thought to be a corruption of the Dutch word "wijd" or "wide," referring to the wide, square-shaped lips of the species. These rhinos are actually gray in color.

**Reply by peterbaccaro** [reply, marked as answer]
The names of black and white rhinos are misleading – as both are actually **grey**.

The white rhino is said to have gotten its name from the Afrikaans word for wide ('wyd'), referring to its wide, square lip (in contrast, black rhinos have a pointy upper lip).

The black rhino probably derives its name as a distinction from the white rhino and from the dark-coloured local soil that covers its skin after wallowing in mud. The upper lip of the black rhino is adapted for feeding from trees and shrubs.
